include/spirv-tools/optimizer.hpp

```diff
@@ -185,6 +185,11 @@ Optimizer::PassToken CreateEliminateDeadConstantPass();
 // points are not changed.
 Optimizer::PassToken CreateInlinePass();
 
+// Creates a compact ids pass.
+// The pass remaps result ids to a compact set which starts from %1 and doesn't
```

Comment:
Prefer "compact range" since I always read "compact set" as "closed and bounded".  Too much topology background I guess. :-)